.gitignore

```diff
@@ -1,3 +1,4 @@
 build_*
 install_*
 source.me
+perfstubs_api/config.h
```

Comment:
This looks  suspicious to me -- why is this needed? If config.h gets created in the source directory, that's a bug that should be fixed. But from looking at `CMakeLists.txt`, it looks to me like it'll correctly get created in the build directory. But then where did it come from?

Reply:
Oops you are correct... either I created it, or it's from an earlier iteration of the build files. I'll remove this from the PR.